src/ducks/account/AccountSwitch.styl

```diff
@@ -50,9 +44,6 @@
 
 baseZ=80
```

Comment:
It seems baseZ is used only at one other place, where it isn't needed now that we are rendered in a Portal. I think we can remove it.